docs/upgrade/xenial_upgrade_in_place.rst

```diff
@@ -0,0 +1,317 @@
+Ubuntu 16.04 LTS (Xenial) - Upgrading in Place
+==============================================
+
+The SecureDrop *Application* and *Monitor* servers may be upgraded-in-place to 
+Xenial using the procedure described here. This procedure does not require 
+physical access to the servers, but it will result in downtime for your instance
+- on the order of 8 hours if no issues are encountered, and longer if anything 
+goes wrong.
+
+The process is less complex than the alternative 
+:doc:`xenial_backup_install_restore` procedure however, and if you are using 
+supported hardware and the downtime described is acceptable, it is the preferred
+option for upgrading your instance.
+
+.. caution::
+  We have tested the upgrade-in-place procedure on officially supported hardware, 
+  but cannot guarantee that the Ubuntu ``do-release-upgrade`` command will 
```

Comment:
This is the first reference to the command `do-release-upgrade`. An admin may not understand this. It should either be referenced in the introductory paragraphs or we should just use a phrase "automated upgrade".

Reply:
changed.